man/systemd-soft-reboot.service.xml

```diff
@@ -122,6 +122,11 @@
     url="https://systemd.io/PORTABLE_SERVICES">Portable Services</ulink>, but make sure no resource from
     the host's root filesystem is pinned via <varname>BindPaths=</varname> or similar unit settings,
     otherwise the old root filesystem will be kept in memory as long as the unit is running.</para>
+
+    <para>If units shall be left running until the very end of shutdown during a soft reboot operation, but
+    shall be terminated regularly during other forms of shutdown, they need to be configured with
+    <varname>Survive=yes</varname>. This will ensure that soft reboot operations do not affect them, but
+    other types of reboot or shutdown stop them as expected.</para>
```

Comment:
Well, they may alternatively use `argv[0][0] = '@'` too.

Reply:
That only helps with the final killing spree though - maybe I could mention it in the 'Survive=' manpage, as a 'beefed up' alternative to that older mechanism?

Reply:
actually that is mentioned above, so I'll update both

Reply:
i think the argv[0][0] thing should work wherever the xattr thing works and vice versa